tests/python/pants_test/backend/project_info/tasks/test_export_dep_as_jar_integration.py

```diff
@@ -118,7 +118,21 @@ def test_javac_options(self):
     self._check_compiler_options_for_target_are(target_to_test, expected_options, config)
 
   def test_extra_jvm_options(self):
-    target_to_test = f'testprojects/src/java/org/pantsbuild/testproject/extra_jvm_options:opts'
+    target_to_test = 'testprojects/src/java/org/pantsbuild/testproject/extra_jvm_options:opts'
     config = {}
     expected_options = {'extra_jvm_options': ['-Dproperty.color=orange', '-Dproperty.size=2', '-DMyFlag', '-Xmx1m']}
     self._check_compiler_options_for_target_are(target_to_test, expected_options, config)
+
+  def test_node_module_deps(self):
+    target_to_test = 'examples/src/scala/org/pantsbuild/example/several_scala_targets:greet_json'
+    export_output = self._run_export_dep_as_jar_goal({}, target_to_test)
+    self.assertIn(
```

Comment:
nit. Could you also assert on the contents of `export_output["libraries"]` here? (in addition to the existence of that field)

Reply:
I'm asserting that the key is present in the libraries dict, do you mean asserting that the path is points to is correct?

Reply:
I think it would be something to the effect of 
```
node_target_exported_path = export_output["libraries"]["examples.src.scala.org.pantsbuild.example.several_scala_targets.node.names_to_greet"]["default"]

assertTrue(os.path.exists(os.path.join(node_target_exported_path, "names.json"))
```